version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.11-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.12-SNAPSHOT"
```

Comment:
No need to adjust the manually, it will be updated automatically on release. It is however needed to change the version number in the `README` manually.